From 009d5cdc0509116c6208f8e606b2395662ea4894 Mon Sep 17 00:00:00 2001
From: Yassine Ilmi <yilmi@vmware.com>
Date: Tue, 25 Feb 2020 10:17:59 +0000
Subject: [PATCH] Adding error management to help identify correct injection

---
 src/public/index.php | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/src/public/index.php b/src/public/index.php
index 6ede760..b684584 100644
--- a/src/public/index.php
+++ b/src/public/index.php
@@ -43,16 +43,20 @@ if (isset($_POST["login"])) {
             $_SESSION["USERID"] = $userId;
             header('Location: /', true , 301);
             exit;
+        } else {
+            print("<b>Authentication failed!</b><br/><br/>");
         }
+    $_SESSION["USERID"] = "";
     } else {
-        // 失敗
+        print("An error occured when querying the LDAP server!");
     }
 }
 
 ?>
 
 <html>
-<?= $_SESSION["USERID"] ?>さん こんにちは
+<? if ($_SESSION["USERID"]) echo '<b>Authentication successful!</b></br></br>' ?>
+Hello! <b><?= $_SESSION["USERID"] ?></b> さん こんにちは
 <form action="/" method="POST">
     <label>User ID: </label><input type="text" name="user_id"/>
     <label>Password: </label><input type="password" name="password"/>
-- 
2.20.1
